win32/devdoc/srw_lock_ll_win32_requirements.md

```diff
@@ -0,0 +1,127 @@
+# `srw_lock_ll_win32` requirements
+
+## Overview
+
+`srw_lock_ll_win32` is the implementation of a slim reader writer lock (`srw_lock_ll` interface). On Windows it simply wraps `SRWLOCK`.
+
+## Exposed API
+
+```c
+#define SRW_LOCK_LL_TRY_ACQUIRE_RESULT_VALUES \
+    SRW_LOCK_LL_TRY_ACQUIRE_OK, \
+    SRW_LOCK_LL_TRY_ACQUIRE_COULD_NOT_ACQUIRE, \
+    SRW_LOCK_LL_TRY_ACQUIRE_INVALID_ARGS
+
+typedef void* SRW_LOCK_LL;
```

Comment:
>void* [](http://example.com/codeflow?start=8&length=5)

Suppose many of the comments from srw_lock_ll_requirements.md apply here too, so Iw ill try to only mention the new ones.

Reply:
"Many of the comments" applied to both